examples/distributed/graphsage/README.md

```diff
@@ -138,3 +138,61 @@ python3 ~/workspace/dgl/tools/launch.py \
 --ip_config ip_config.txt \
 "python3 node_classification.py --graph_name ogbn-products --ip_config ip_config.txt --num_epochs 30 --batch_size 1000 --num_gpus 4"
 ```
+
+### Running with GraphBolt
+
+In order to run with `GraphBolt`, we need to partition graph into `GraphBolt` data formats.Please note that both `DGL` and `GraphBolt` partitions are saved together.
+
+```
+python3 partition_graph.py --dataset ogbn-products --num_parts 2 --balance_train --balance_edges --use_graphbolt
+```
+
+#### Partition sizes compared to DGL
+
+Compared to `DGL`, `GraphBolt` partitions are much smaller(reduced to **16%** and **19%** for `ogbn-products` and `ogbn-papers100M` respectively).
+
+`ogbn-products`
+
+| Data Formats |         File Name            | Part 0 | Part 1 |
+| ------------ | ---------------------------- | ------ | ------ |
+| DGL          | graph.dgl                    | 1.5GB  | 1.6GB  |
+| GraphBolt    | fused_csc_sampling_graph.pt  | 255MB  | 265MB  |
+
+`ogbn-papers100M`
+
+| Data Formats |         File Name            | Part 0 | Part 1 |
+| ------------ | ---------------------------- | ------ | ------ |
+| DGL          | graph.dgl                    | 23GB   | 22GB   |
+| GraphBolt    | fused_csc_sampling_graph.pt  | 4.4GB  | 4.1GB  |
+
+Then run example with `--use_graphbolt`.
+
+```
+python3 ~/workspace/dgl/tools/launch.py \
+--workspace ~/workspace/dgl/examples/pytorch/graphsage/dist/ \
+--num_trainers 4 \
+--num_samplers 0 \
+--num_servers 2 \
+--part_config data/ogbn-products.json \
+--ip_config ip_config.txt \
+"python3 node_classification.py --graph_name ogbn-products --ip_config ip_config.txt --num_epochs 10 --use_graphbolt"
+```
+
+#### Performance compared to `DGL`
```

Comment:
What about the peak memory?

Reply:
peak mem of what? loaded graph partition? ram usage during sampling or the whole training?